lib/logitech_receiver/descriptors.py

```diff
@@ -213,7 +213,14 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Mice
 
 _D('Wireless Mouse M175', protocol=2.0, wpid='4008')
-_D('Wireless Mouse M185')
+_D('Wireless Mouse M185 new', codename='M185n', protocol=4.5, wpid='4054',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
+_D('Wireless Mouse M185 old', codename='M185o', protocol=4.5, wpid='4055',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
```

Comment:
Hello @jrbenito 
Thank you for your input. Looks like Logitech massively re-utilize model numbers for different devices, and model name can't be used for device feature set detection. I'm think we should remove settings parameter from descriptors and totally rely on feature auto-detection. But, I remember i encounter some problems trying to use auto-detect for feature_toggle template function.  I should investigate this in details then have some spare time.